src/vs/workbench/contrib/terminalContrib/chatAgentTools/common/terminalChatAgentToolsConfiguration.ts

```diff
@@ -296,6 +298,19 @@ export const terminalChatAgentToolsConfiguration: IStringDictionary<IConfigurati
 		minimum: -1,
 		maximum: 60000,
 		default: -1
+	},
+	[TerminalChatAgentToolsSettingId.AutoReplyToPrompts]: {
```

Comment:
This should be tagged with experimental

Reply:
@digitarald anything else we need for experimental settings that would eventually be controlled via policy?

Reply:
Nothing else needed prior to adding the `policy` object (which sounds like will come later?)